src/System.Management.Automation/engine/PathInterfaces.cs

```diff
@@ -1122,15 +1122,15 @@ internal string Combine(string parent, string child, CmdletProviderContext conte
         /// <exception cref="ProviderInvocationException">
         /// If the provider threw an exception.
         /// </exception>
-        public string ParseParent(string path, string root)
+        public string ParseParent(string path, int depth, string root)
```

Comment:
We shouldn't change a public API. Please revert the change and add new overload.

Reply:
Otherwise, this is a breaking change...